### PR TITLE
Fix Broken Links close #68

### LINKED ISSUE #68: Some links in Readme give 404
## These are the dead links in the readme file.

GeckoView Contributor Quick Start Guide
Mozilla Central Quick Start Guide
Mozilla Central Contributor Guide
Guide to Native Debugging in Android Studio

### PR DESCRIPTION
**Pull request by strongSoda**
fix for #68

### CODE REVIEW

**Comment by strongSoda** [comment]
@agi90  all the links work fine now.

File: README.md [hunk 1]
```diff
@@ -28,13 +28,14 @@ You can read more about GeckoView on the [wiki](https://wiki.mozilla.org/Mobile/
 
 
 [1]:https://geckoview.dev/consumer/docs/geckoview-quick-start
-[2]:https://geckoview.dev/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG.md
-[3]:https://geckoview.dev/contributor/geckoview-quick-start.md
-[4]:https://geckoview.dev/contributor/mc-quick-start.md
-[5]:https://geckoview.dev/contributor/contributing-to-mc.md
-[6]:https://geckoview.dev/contributor/native-debugging.md
-[7]:https://geckoview.dev/consumer/docs/web-extensions.md
+[2]:https://mozilla.github.io/geckoview/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG
```
Comment (agi90) [inline README.md]: these should still link to `geckoview.dev`:  `https://geckoview.dev/javadoc/mozilla-central/org/mozilla/geckoview/doc-files/CHANGELOG`

(applies to all of them)

**Review by agi90 — COMMENTED** [review]
Thank you! Just one comment before we can merge this

**Comment by strongSoda** [comment]
@agi90 I have made the changes. Also, the API documentation link pointed to a GitHub file, I made it to point to https://geckoview.dev/javadoc/mozilla-central/index.html

**Comment by agi90** [comment]
Thank you!

**Pull request merged**